internal/it/map_tester.go

```diff
@@ -9,18 +9,25 @@ import (
 	"github.com/hazelcast/hazelcast-commandline-client/internal/check"
 )
 
-func WithMap(tcx TestContext, fn func(m *hz.Map)) {
-	name := NewUniqueObjectName("map")
+func WithMap(tcx TestContext, mn string, fn func(m *hz.Map)) {
```

Comment:
How about using a different name for this function instead of renaming the other. Maybe `WithNamedMap` ?